ZelBack/src/services/fluxCommunication.js

```diff
@@ -1258,7 +1263,7 @@ async function checkMyFluxAvailability(retryNumber = 0) {
   }
   let askingIpPort = config.server.apiport;
   if (askingIP.includes(':')) { // has port specification
-    // it is ipv6
```

Comment:
How does this work for IPv6 and :: in the IP portion

Reply:
Flux is not compatible with ipv6, obsolete comment removal